R/plotSummary.R

```diff
@@ -146,6 +169,7 @@ plotSummary <- function(path, facet = "typ", showHistStock = FALSE) {
     pOut <- pOut +
       scale_fill_manual(values = fillColours[[fillDim]],
                         labels = fillLabels[[fillDim]]) +
+      scale_alpha_manual(values = c(`FALSE` = 1, `TRUE` = 0.2), guide = "none") +
```

Comment:
For my IEW abstract, Robert remarked that the transparent bars are maybe a bit too transparent. I think I set this to 0.3 instead of 0.2 - maybe you can have a look which level of transparency you prefer.

Reply:
Sounds good. Let's go with 0.3 then.